### PR TITLE
Remove CodePen link in Intermediate CSS - CSS Functions

### LINKED ISSUE #23404: Improperly embedded CodePen Edit Link in INTERMEDIATE CSS - CSS Functions
Screen Shot 2022-01-26 at 10 28 07 PM

There is a link to Edit a CodePen in the CSS Functions lesson of INTERMEDIATE CSS that links to an Imgur image. I'm not sure if the fix would be as simple as moving the Embedded CodePen from above down to where this link is or if it is better have an external link to the CodePen.

### PR DESCRIPTION
**Pull request by gregolive**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Removed the Edit CodePen link in the CSS Functions lesson of INTERMEDIATE CSS. Edited line 60 to instruct the reader to refer to the embedded CodePen above and moved the paragraph up to line 58.

#### 2. Related Issue

Closes #23404

### CODE REVIEW

File: html_css/intermediate_css/css_math_functions.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ To put it another way:  `main = 100vh - (header + footer)`.
 `calc()` is handling the math for us even though we are mixing vh, rem and px units.
 Combined with CSS variables, `calc()` can save us from the headache of repeating CSS rules.
 
-**Note:** The above is just an example of how `calc()` can affect a layout, but keep in mind that `calc()` is likely not the best way to go about it. We will talk more about layouts in future lessons.
+I encourage you to take a moment and edit the CodePen above to understand how `calc()` is being used. Play around with the different units and sizes of the elements and see what results you get before moving on.
```
Comment (01zulfi) [inline html_css/intermediate_css/css_math_functions.md:58]: Wording suggestion:

```suggestion
You should be able to grasp how `calc()` is used in the above CodePen embed. We encourage you to play around with different units and sizes of the elements to see what results you get before moving on.
```

**Review by 01zulfi — CHANGES_REQUESTED** [review]

**Review by gregolive — COMMENTED** [review]
@01zulfi Updated (I hope) with your suggested wording.

**Review by 01zulfi — APPROVED** [review]
Looks good :rocket: 

Thanks for contributing!

**Comment by gregolive** [comment]
> Looks good 🚀
> 
> Thanks for contributing!

Thank you for the help!

**Pull request merged**